### DISCUSSION TITLE
I want to update rotation, offset, scale, wrapS and wrapT using api

### DISCUSSION THREAD

**Discussion by kalpeshpolynine** [google/model-viewer#4286] [Q&A]
Hello @elalish,

I want to use update model rotation, offset, scale, wrapS and wrapT using api but none of them are working for me. I have refer this documentation: https://modelviewer.dev/examples/scenegraph/#transformTexturesExample

This is my example link 
https://customer.polynine.com/poc/suitepieces/test_sampler.html

It's not working and showing  this error 

<img width="1920" alt="image" src="https://github.com/google/model-viewer/assets/58295658/73205642-6ae2-45a6-af5d-acbfee957a5f">

Here are reference documents:
https://github.com/google/model-viewer/issues/3368

Would you please help to solve this? It will be a big unlock for me. 

Code: 

**Model viewer Library**

`<script type="module" src="https://ajax.googleapis.com/ajax/libs/model-viewer/3.1.1/model-viewer.min.js"></script>`

<img width="1064" alt="image" src="https://github.com/google/model-viewer/assets/58295658/bd8ae4ff-da01-4e66-ab3c-ebb861be7564">

**Comment by elalish** [comment]
Sorry, that's pre-release: https://modelviewer.dev/docs/faq.html#entrydocs-general-questions-api

**Reply by kalpeshpolynine** [reply]
I want to use pre-release on my current project. How do I do that?

**Reply by elalish** [reply]
Did you read the link?
